src/checkout-buttons/checkout-button-error-selector.ts

```diff
@@ -0,0 +1,24 @@
+import { InternalCheckoutSelectors } from '../checkout';
+import { selector } from '../common/selector';
+
+import CheckoutButtonSelector from './checkout-button-selector';
+
+@selector
+export default class CheckoutButtonErrorSelector {
+    private _checkoutButton: CheckoutButtonSelector;
+
+    /**
+     * @internal
+     */
+    constructor(selectors: InternalCheckoutSelectors) {
+        this._checkoutButton = selectors.checkoutButton;
+    }
+
+    getInitializeButtonError(methodId?: string): Error | undefined {
```

Comment:
should you document these methods (what happens if methodId is falsy?)

Reply:
Yeah good point. I haven't documented anything yet.

Reply:
The documentation might have to come as a separate PR.